x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/components/curations_table.tsx

```diff
@@ -101,17 +102,29 @@ export const CurationsTable: React.FC = () => {
   ];
 
   return (
-    <EuiBasicTable
-      columns={columns}
-      items={curations}
-      responsive
-      hasActions
-      loading={dataLoading}
-      pagination={{
-        ...convertMetaToPagination(meta),
-        hidePerPageOptions: true,
-      }}
-      onChange={handlePageChange(onPaginate)}
-    />
+    <DataPanel
```

Comment:
I'm using `DataPanel` for both this new table AND I've updated the curations table to use it as well.

It seems to match exactly what the designs call for and it keep our code consistent.